backend/server/web.go

```diff
@@ -34,7 +36,7 @@ type WebConfig struct {
 	DBConnection     *database.Connection
 	AuthSchemes      []authschemes.AuthScheme
 	CSRFAuthKey      []byte
-	SessionStoreKey  []byte
+	SessionStoreKey  string
```

Comment:
I repurposed the sessionstorekey to be the key name for the user sessions (the session is saved to each user's context, so I think naming it all the same thing shouldn't be a problem?), and the key must be a string